processor/attributesprocessor/attribute_hasher.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package attributesprocessor
+
+import (
+	"crypto/sha1"
+	"encoding/binary"
+	"encoding/hex"
+	"math"
+
+	tracepb "github.com/census-instrumentation/opencensus-proto/gen-go/trace/v1"
+)
+
+const (
+	maxVarFloat64 = 8
```

Comment:
nit: same suggestion to the one below
```suggestion
	float64ByteSize = 8
```